src/app/@ansyn/ansyn/modules/filters/components/slider-filter-container/slider-filter-container.component.ts

```diff
@@ -33,7 +33,7 @@ export class SliderFilterContainerComponent {
 	@Input() counters: SliderFilterCounters;
 
 	@HostBinding('class.rtl')
-	@Input() isRTL = false;
+	@Input() isRTL = true;
```

Comment:
remove instances of @Input() isRTL = true;

Reply:
Done